dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -58,23 +62,23 @@ def create_glove_char(n_dims, source_file=None):
 
     # get PCA embedder
     pca = decomposition.PCA(n_components=n_dims)
-    reduced_embds = pca.fit_transform(embd_matrix)
+    reduced_embds: np.ndarray = pca.fit_transform(embd_matrix)
 
     # write to file
     dir_name = os.path.dirname(source_file)
     embd_file_name = os.path.join(dir_name, "glove-reduced-{}D.txt".format(n_dims))
     with open(embd_file_name, "w") as file:
-        for word, embd in zip(embd_words, reduced_embds):
+        for word, embd in zip(embd_words, reduced_embds):  # type: ignore
```

Comment:
why the ignore?

Reply:
I just added typing to `embd_words`, so its unnecessary now